src/status_im/ui2/screens/common/contact_list/view.cljs

```diff
@@ -9,10 +9,12 @@
   [quo/divider-label {:label title}])
 
 (defn contact-list
-  [data]
-  (let [contacts (if (:group data)
-                   (rf/sub [:contacts/add-members-sections])
-                   (rf/sub [:contacts/filtered-active-sections]))]
+  [{:keys [start-a-new-chat?] :as data}]
+  (let [contacts (if start-a-new-chat?
```

Comment:
A new subscription for this logic seems better to me and could even lead to better performance. Notice that `contact-list` will needlessly re-render due to re-frame registering 2-3 different *reactions*. A single, new subscription *could* reduce the rendering overhead.